README.md

```diff
@@ -96,6 +128,14 @@ Before committing, please lint your code:
 make lint
 ```
 
+And make sure the tests pass:
+
+```sh
+make test
```

Comment:
Do you know why `test` is syntax highlighted? It's not a problem, but it's kind of weird and I have no idea why it would do that for this one and not the other examples above

Reply:
Looks like some weirdness with GitHub flavored markdown. I removed the `sh` hint from the make commands since it doesn't do anything for us, anyway.